Assets/Scripts/Game/UserInterfaceWindows/DaggerfallWitchesCovenPopupWindow.cs

```diff
@@ -131,10 +131,11 @@ protected override void GetQuest()
             // Get the faction id for affecting reputation on success/failure, and current rep
             int factionId = witchNPC.Data.factionID;
             int reputation = GameManager.Instance.PlayerEntity.FactionData.GetReputation(factionId);
+            int rank = 0;
             int level = GameManager.Instance.PlayerEntity.Level;     // Not a proper guild so rank = player level
```

Comment:
Just noticed that I broke the witches coven, that so far intentionally sent the level as rank parameter.
Either we keep doing that, or add the questByRankOrLevel to them so they enter the PR framework